extensions/amazon-sales/release-notes/index.md

```diff
@@ -3,19 +3,29 @@ group: extensions
 title: Amazon Sales Channel Release Notes
 ---
 
-**Amazon Sales Channel** is now available for version 2.2.x and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell products in your Magento catalog on Amazon.
+**Amazon Sales Channel** is available for versions 2.2.x and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}. Merchants can use Amazon Sales Channel to integrate with an Amazon Seller account to post and sell products in your Magento catalog on Amazon.
 
 See the following documentation:
 
-- [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
-- [Amazon Sales Channel install]({{site.baseurl}}/extensions/amazon-sales/) for installation and API key information
-- [Amazon Sales Channel Marketplace](http://marketplace.magento.com/magento-module-amazon.html)
+-  [Amazon Sales Channel](https://docs.magento.com/m2/ce/user_guide/sales-channels/amazon/amazon-sales-channel.html) for merchant information and instructions
+-  [Amazon Sales Channel install]({{site.baseurl}}/extensions/amazon-sales/) for installation and API key information
+-  [Amazon Sales Channel Marketplace](http://marketplace.magento.com/magento-module-amazon.html)
 
 The release notes include:
 
--   {:.new}New features
--   {:.fix}Fixes and improvements
--   {:.bug}Known issues
+-  {:.new}New features
+-  {:.fix}Fixes and improvements
+-  {:.bug}Known issues
+
+### v2.0.1
+
+Amazon Sales Channel 2.0.1 is compatible with versions 2.2.4+ and 2.3.x of {{site.data.var.ce}}, {{site.data.var.ee}}, and {{site.data.var.ece}}.
+
+-  {:.new}**Improved Logging**: <!--CHAN-3642, 3672-->Implemented **Enable Debug Logging** feature to collect additional synchronization data when troubleshooting is needed. See the [Sales Channels Settings](https://docs.magento.com/m2/ce/user_guide/configuration/sales-channels/global-settings.html) topic in the Configuration Reference.
+
+-  {:.fix}**Product Catalog**: <!--CHAN-3687-->Corrected an issue preventing images imported from an Amazon listing from being applied to the corresponding Magento catalog product.
+
+-  {:.fix}**Order Creation**: <!--CHAN-3708-->Corrected an issue preventing Magento from creating orders for an Amazon order that does not match with a Magento catalog product. See [Order Settings](https://docs.magento.com/m2/ee/user_guide/sales-channels/amazon/order-settings.html).
```

Comment:
This link goes to the Commerce version, while the first bulleted item goes to the Open Source. Make them consistent, or state that this fix applies to EE only.